web/css/components/text-icon.css

```diff
@@ -0,0 +1,29 @@
+body {
```

Comment:
delete style applied to body, if you need to set somes, set the style in the html for demo purpose